iXBRLViewerPlugin/viewer/src/js/util.js

```diff
@@ -149,3 +149,14 @@ export function titleCase(text) {
                 .join('');
     }).join(' ');
 }
+
+/**
+ * Extract the alpha value from an rgba(r,g,b,a) string and returns true if
+ * transparent (a < 10%).  Returns false on rgb(r,g,b) strings.
+ * @param  {String}  rgba   color string to parse
+ * @return {Boolean} whether the value is transparent
+ */
+export function isTransparent(rgba) {
+    const val = parseFloat(rgba.split(",")[3]);
+    return !isNaN(val) && val < 0.1;
```

Comment:
I guess the `isNaN` check is redundant, but it's perhaps clearer to be explicit?

Reply:
I hadn't spotted that it's redundant, but I do think it is worth being explicit, as it documents that `NaN` is expected on some inputs, and makes it clear how it's handled.